### PR TITLE
Adding a new test for verification of the default csi subvolume group pinning value. 

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR contains a new test for verification of the default csi subvolume group pinning value. 
This is an automation of RHSTOR-3270- Enable distributed ephemeral pins on "csi" subvolume group.

### CODE REVIEW

**Review by ramkiperiy — DISMISSED** [review]

File: tests/functional/storageclass/test_csi_subvolume_group_property.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            err_msg = 'Pinning property not found, missing key "%s"' % str(e)
+            logger.error(err_msg)
+            raise Exception(err_msg)
+
+        if pinning_val != 1:
+            err_msg = f"Expected pinning value 1, got {pinning_val} instead"
+            logger.error(err_msg)
+            raise Exception(err_msg)
+
+        logger.info("Pinning value found and it is 1, ax expected")
```
Comment (keesturam) [inline tests/functional/storageclass/test_csi_subvolume_group_property.py:34]: `ax` should be `as`

File: tests/functional/storageclass/test_csi_subvolume_group_property.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from ocs_ci.helpers.performance_lib import run_oc_command
+
+logger = logging.getLogger(__name__)
+
+
+@green_squad
+@skipif_external_mode
+@tier3
+class TestCSISubvolumeGroup(ManageTest):
+    def test_network_fence_not_editable(self):
```
Comment (keesturam) [inline tests/functional/storageclass/test_csi_subvolume_group_property.py:15]: The test name doesn't seem to match with the flow of the test. any reason as to why the name is `test_network_fence_not_editable`?

File: tests/functional/storageclass/test_csi_subvolume_group_property.py (hunk 1 above)
Comment (ypersky1980) [reply tests/functional/storageclass/test_csi_subvolume_group_property.py:34]: Fixed.

File: tests/functional/storageclass/test_csi_subvolume_group_property.py (hunk 2 above)
Comment (ypersky1980) [reply tests/functional/storageclass/test_csi_subvolume_group_property.py:15]: Changed to a meaningful name , of course that was a typo that should have been fixed.

**Review by ramkiperiy — APPROVED** [review]

**Review by keemano — APPROVED** [review]

**Pull request merged**